pkg/app/collector.go

```diff
@@ -116,6 +116,7 @@ func (a *App) StartCollector(ctx context.Context) {
 						return
 					}
 				case tErr := <-errChan:
+					subscribeResponseFailedCounter.WithLabelValues(t.Config.Name, tErr.SubscriptionName).Inc()
```

Comment:
Let's move the `.Inc()` into the `else` statement. `EOF` is not a real error, it's how a server closes a stream.

Reply:
done